tests/tfm/tfm_psa_test/README.rst

```diff
@@ -12,19 +12,18 @@ The TF-M platform security architecture test sample provides a basis for validat
 Requirements
 ************
 
-When the Kconfig option :kconfig:option:`CONFIG_TFM_PSA_TEST_ATTESTATION` is enabled, it is required that the device is provisioned with the PSA root-of-trust security parameters using the :ref:`provisioning image <provisioning_image>` sample.
-To provision the device, build and flash the provisioning image sample before using the test sample.
-
 The test supports the following development kits:
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :rows: nrf5340dk_nrf5340_cpuapp_ns, nrf9160dk_nrf9160_ns
+   :rows: nrf5340dk_nrf5340_cpuapp_ns, nrf9160dk_nrf9160_ns, nrf9151dk_nrf9151_ns
 
 Overview
 ********
 
-The PSA tests are implemented in the psa-arch-tests repo: https://github.com/ARM-software/psa-arch-tests.
+The PSA tests are implemented in the psa-arch-tests repository: https://github.com/ARM-software/psa-arch-tests.
+Tests can be found under :file:`/modules/tee/tf-m/psa-arch-tests` in the |NCS| folder structure.
```

Comment:
This folder is actually checked out from https://github.com/nrfconnect/sdk-psa-arch-tests which is a fork of the mentioned repository. I'm not sure what confused people more to mention here in the doc that there the actual tests are a fork, or not saying anything in the doc and somebody checking their remotes in that folder one day.

Reply:
How about changing it to something like this:
```suggestion
Tests can be found under :file:`/modules/tee/tf-m/psa-arch-tests` in the |NCS| folder structure.
They are checked out from `nrfconnect/sdk-psa-arch-tests <https://github.com/nrfconnect/sdk-psa-arch-tests>`_, which is forked from `zephyrproject-rtos/psa-arch-tests <https://github.com/zephyrproject-rtos/psa-arch-tests>`_, which in turn is forked from the source repository: `ARM-software/psa-arch-tests <https://github.com/ARM-software/psa-arch-tests>`_.

```

Reply:
Sounds perfect for me :)